core_lib/src/structure/layercamera.cpp

```diff
@@ -166,13 +210,13 @@ void LayerCamera::setViewRect(QRect newViewRect)
     emit resolutionChanged();
 }
 
-void LayerCamera::loadImageAtFrame(int frameNumber, qreal dx, qreal dy, qreal rotate, qreal scale)
+void LayerCamera::loadImageAtFrame(int frameNumber, qreal dx, qreal dy, qreal rotate, qreal scale, int easing)
```

Comment:
Like the other classes I'd prefer if we could keep using the CameraEasingType enum type here instead of int